src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+
+function Articles({articles, setArticles}){
+
+  const [title, setTitle] = useState('')
+
+    useEffect(() => {
+        axios
+        .get('https://northcoders-news-app-ei5k.onrender.com/api/articles')
+        .then((response) => {
+            setArticles(response.data.articles)
```

Comment:
I don't see why you're making an api call to send info out of this component and back down to this component. The articles and setArticles should be in this component.